vendor/github.com/shirou/gopsutil/v3/disk/disk_openbsd.go:59:53: cannot use stat.F_mntfromname[:] (value of type []byte) as type []int8 in argument to common.IntToString                                                                      
vendor/github.com/shirou/gopsutil/v3/disk/disk_openbsd.go:60:51: cannot use stat.F_mntonname[:] (value of type []byte) as type []int8 in argument to common.IntToString                                                                        
vendor/github.com/shirou/gopsutil/v3/disk/disk_openbsd.go:61:52: cannot use stat.F_fstypename[:] (value of type []byte) as type []int8 in argument to common.IntToString                                                                       
vendor/github.com/shirou/gopsutil/v3/disk/disk_openbsd.go:149:45: cannot use stat.F_fstypename[:] (value of type []byte) as type []int8 in argument to common.IntToString

Index: vendor/github.com/shirou/gopsutil/v3/disk/disk_openbsd.go
--- vendor/github.com/shirou/gopsutil/v3/disk/disk_openbsd.go.orig
+++ vendor/github.com/shirou/gopsutil/v3/disk/disk_openbsd.go
@@ -56,9 +56,9 @@ func PartitionsWithContext(ctx context.Context, all bo
 		}
 
 		d := PartitionStat{
-			Device:     common.IntToString(stat.F_mntfromname[:]),
-			Mountpoint: common.IntToString(stat.F_mntonname[:]),
-			Fstype:     common.IntToString(stat.F_fstypename[:]),
+			Device:     common.ByteToString(stat.F_mntfromname[:]),
+			Mountpoint: common.ByteToString(stat.F_mntonname[:]),
+			Fstype:     common.ByteToString(stat.F_fstypename[:]),
 			Opts:       opts,
 		}
 
@@ -146,7 +146,7 @@ func UsageWithContext(ctx context.Context, path string
 }
 
 func getFsType(stat unix.Statfs_t) string {
-	return common.IntToString(stat.F_fstypename[:])
+	return common.ByteToString(stat.F_fstypename[:])
 }
 
 func SerialNumberWithContext(ctx context.Context, name string) (string, error) {
